### ISSUE TITLE
Side menu

### ISSUE THREAD

**Issue by CMS-F** [ChristopherDosin/Shopware-6-Blog-Plugin#91]
Hi,
as discussed by e-mail Part 2:
In the original plugin, two dropdown fields are displayed as filters at the top of the listing page. I would like to have a side menu on the left side on the listing page and on the detail page. This should display the categories alphabetically sorted below each other. The appearance should be similar to the category overview for shop-categories.

Hallo,
wie per E-Mail besprochen hier Teil 2:
Derzeit gibt es nur zwei Dropdown-Felder zur Auswahl bzw. als Filter auf der Übersichtsseite. Es wäre schön, wenn die Kategorien auf der linken Seite alphabetisch dargestellt würden (sowohl auf der Übersichtsseite als auch auf der Detailseite). Optisch sollte dieses ähnlich wie die Kategorieauswahl bei Artikeln sein, damit es optisch im Shop zusammenpasst.

**Comment by ChristopherDosin** [comment]
@viennt A good option would be maybe that we split the actual listing element & the filters. So we would have just one blog listing element - And then a separate blog filter element. So the user is able to move the filters wherever he wants - Kinda like the same as for the product-listing filters.

**Comment by viennt** [comment]
@ChristopherDosin Agree.
But I guess the categories order should be based on the order in administration?
I also prepare a draft mockup for this element. Please take a look.

Blog Filter Element Mockup:
<img width="320" alt="Screen Shot 2022-04-22 at 15 06 52" src="https://user-images.githubusercontent.com/7336695/164645676-29eaa0bb-1455-473b-a62a-cbff0f81eb7f.png">
 
Administration Page:
<img width="614" alt="Screen Shot 2022-04-22 at 15 16 20" src="https://user-images.githubusercontent.com/7336695/164647333-6af0f115-2f70-422d-89a2-dce3fb47f913.png">

**Comment by ChristopherDosin** [comment]
Yes - The category order should be the same as within the administration. Thanks for the Mockup 👍 
I guess we should maybe also split the categories & filters in it's own block+element.

So we're giving here the full flexibility where the user wants to have the categories and/or the filters.

**Comment by viennt** [comment]
So what we'll do are:

1. Update `blog-listing` element: remove categories & filter
2. Add new `blog-category` element
3. Add new `blog-filter` element

Is that correct?

**Comment by ChristopherDosin** [comment]
Correct. But not only we need the elements, also the new Block for the `blog-category` and `blog-filter` element.

**Comment by viennt** [comment]
Yes, I know ^^

**Comment by CMS-F** [comment]
Hello,
Thank you very much for including me in the conversation. I think the solution is very good and I am looking forward to the result.
A really great service
Many greetings

Christian
